rpm/pext.spec

```diff
@@ -0,0 +1,44 @@
+#Disable debug packages
+%define debug_package %{nil}
+
+Name:	pext
+Version:	0.9
+Release:	1%{?dist}
+Summary:	Python-based extendable tool
+
+License:	GPLv3
+URL:		https://github.com/Pext/Pext
+Source0: 	https://github.com/Pext/Pext/archive/v%{version}.zip
+#This removes the dependency since dependency checks are done by dnf/yum/rpm.
+#Also in Fedora the module name is PyQt5
+Patch0:		remove-dependency.patch
+
+BuildRequires:	python3-devel
+Requires:	python3-qt5 python3-pyopengl
+
+%description
+Pext stands for Python-based extendable tool. It is built using Python 3 and Qt5 QML and has its behaviour decided by modules. Pext provides a simple window with a search bar, allowing modules to define what data is shown and how it is manipulated.
+
+%prep
+%setup -q -n Pext-%{version}
+%patch0
+
+%build
+python3 setup.py build
+
+%install
+python3 setup.py install --prefix=%{_prefix} --root=%{buildroot}
+
+%files
```

Comment:
Could you explain how I would know what files should be added to %files and which not? https://fedoraproject.org/wiki/How_to_create_an_RPM_package#.25files_section seems to just state "This section declares which files and directories are owned by the package, and thus which files and directories will be placed into the binary RPM." which makes me wonder... what about all the QML files?

Reply:
My typical method for retrieving the list of files (may be a better way) is to build the RPM with  an empty file list. rpmbuild will then output the list of files that are in the buildroot that are not in the files section. After that I take the file list and substitute macros where appropriate along with tags for configuration files if there are any.

As for the QML files, since they get placed in to the buildroot during the setup install, they automatically get captured by the ```%{python3_sitelib}/*``` glob which gets expanded to all of the files in the buildroot under ```/usr/lib/python3.6/site-packages/```.

Does that answer your question?

Reply:
Okay, yes, that I understand. Should be enough for me to be able to maintain this list if I ever have to make changes. Thanks!